### ISSUE TITLE
training loss is nan

### ISSUE THREAD

**Issue by godwardad** [DawoonJang/FCN_Pytorch#1]
I am training "ConvolutionalVGGwithUpsample" model on VOC seg  and getting loss nan after few iterations. tried with different losses. please provide your dataset and training code.
